src/components/utils/test/posix_thread_test.cc

```diff
@@ -41,287 +43,466 @@ namespace utils_test {
 using namespace sync_primitives;
 using namespace threads;
 
-// TODO(AByzhynar): Change this to use Gtest class to create all variables for
-// every TEST_F
-// TODO(AByzhynar): Add multithreading tests
-
 namespace {
-const uint32_t MAX_SIZE = 20;
-const size_t MyStackSize = 32768;
-const char* threadName("test thread");
-const std::string test_thread_name("THREAD");
-sync_primitives::ConditionalVariable cond_var_;
-sync_primitives::Lock test_mutex_;
+const uint32_t kMaxSize = 20;
+const size_t kStackSize = 32768;
+const char* kThreadName = "test thread";
+const std::string kTestThreadName("THREAD");
```

Comment:
Maybe it is enough to use `kTestThreadName.c_str()` instead of `kThreadName`? Or this was done for thread-safe meanings?

Reply:
@YarikMamykin 
Fixed in:https://github.com/LuxoftSDL/sdl_core/pull/50/commits/ca62978ee665d27ecfceffac8caf96ce740ecb51